blockly-core/core/ui/contract_editor/contract_editor.js

```diff
@@ -91,16 +101,15 @@ goog.inherits(Blockly.ContractEditor, Blockly.FunctionEditor);
 Blockly.ContractEditor.EXAMPLE_BLOCK_TYPE = 'functional_example';
 Blockly.ContractEditor.EXAMPLE_BLOCK_ACTUAL_INPUT_NAME = 'ACTUAL';
 
-Blockly.ContractEditor.typesToColorsHSV = {
-  // 'none': [0, 0, 0.6], // not available to users
-  'Number': [192, 1.00, 0.99], // 00ccff
-  'string': [180, 1.00, 0.60], // 0099999
-  'image': [285, 1.00, 0.80], // 9900cc
-  'boolean': [90, 1.00, 0.4] // 336600
-};
+Blockly.ContractEditor.typesToColorsHSV = goog.object.filter(
```

Comment:
This seems complex enough that it deserves a comment. Is this essentially just duplicating FunctionalTypeColors, but without 'none'? Is there a downside to have none in this object too, and depending on USER_TYPE_CHOICES to disallow it? Alternatively, could we just clone FunctionalTypeColors and then explicitly set none to undefined (assuming I'm right that this is the only difference).